### PR TITLE
added pg_proctab functions

### PR DESCRIPTION
**Pull request by davecramer**
This is the easy way.  Just copied the code from pg_proctab in and modified the makefile. Future work to make it more like pgnodemx to use the parsing functions in pgnodemx

### CODE REVIEW

File: parseutils.c [hunk 1]
```diff
@@ -542,6 +542,8 @@ parse_keqv_line(char *line)
 	return values;
 }
 
+
```
Comment (jconway) [inline parseutils.c]: gratuitous white-space addition

File: pg_proctab.c [hunk 2]
```diff
@@ -16,20 +16,54 @@
 #include <sys/stat.h>
```
Comment (jconway) [inline pg_proctab.c]: Admittedly pgnodemx.c is getting a bit too long, but in any case the other "proc" functions are all defined in there. I think for this first addition of "proctab" functions we should keep everything in pgnodemx.c for the sake of consistency. A later cleanup might be good which moves all of the "proc" functions to their own source file (perhaps procfuncs.c?).

File: parseutils.c (hunk 1 above)
Comment (davecramer) [reply parseutils.c]: ya, will deal with this, thanks for reviewing

File: pg_proctab.c (hunk 2 above)
Comment (davecramer) [reply pg_proctab.c]: Fair enough. Once it works I can move it back into pgnodemx.c

File: genutils.c [hunk 3]
```diff
@@ -745,3 +747,19 @@ pg_ulltoa_n(uint64 value, char *a)
...
+/*
+ * Convert number of kernel pages to size in bytes
+ */
+PG_FUNCTION_INFO_V1(pgnodemx_pages_to_bytes);
+Datum
+pgnodemx_pages_to_bytes(PG_FUNCTION_ARGS)
+{
+	Numeric		num1 = PG_GETARG_NUMERIC(0);
+	Numeric		num2  = int64_to_numeric(sysconf(_SC_PAGESIZE));
+	Numeric		res;
```
Comment (davecramer) [inline genutils.c:760]: too many spaces. tab issues ?

File: parseutils.c [hunk 4]
```diff
@@ -623,3 +623,43 @@ parse_space_sep_val_file(char *ftr, int *nvals)
...
+{
+	char **lines = read_nlsv(fname, nlines);	
+
+	if (nlines > 0)
+	{
+		char ***values;
+		int		nrow = *nlines;
+		int		ncol = 2;
+		int		i;
+
```
Comment (davecramer) [inline parseutils.c:648]: similar comment to spacing ?

**Review by jconway — APPROVED** [review]

**Pull request merged**